CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # bedrock-resource-restriction ChangeLog
 
+## 3.0.0 - 2021-XX-XX
+
+### Changed
+- **BREAKING**: `restrictions.get` now returns the `restrictions` property
+  with an array of all restrictions matching the `zone` and `resource`.
+
+### Added
+- Added `getById` and `removeById` to `restrictions`.
```

Comment:
Since this is a breaking change, we can avoid breaking convention with our other libs here. That means we should make `get` take `id` and return a single restriction and add `getAll` and have it return an array based on taking `zone` and `resource`.